package/src/Slider.tsx

```diff
@@ -285,7 +285,7 @@ SliderWithRef.defaultProps = {
 };
 
 let styles = StyleSheet.create(
-  Platform.OS === 'ios' ? {slider: {height: 40}} : {slider: {}},
+  Platform.OS === 'ios' ? {slider: {}} : {slider: {}},
```

Comment:
With removing the `height: 40` style, this line does actually nothing, and could even be simplified into:
```suggestion
  {slider: {}}
```